lib/service_actor/result.rb

```diff
@@ -61,13 +65,13 @@ def []=(key, value)
     data[key] = value
   end
 
-  def delete(key)
+  def __delete__(key)
```

Comment:
I’d rather not use the `__*__` syntax which is very linked to Ruby internals.

Another way we could name it would be `delete!` since inputs and outputs would not make sense using special characters. It also pairs well with `merge!`.

Reply:
Agreed, renamed to `delete!`